pkg/wallet/risk.go

```diff
@@ -45,42 +45,54 @@ func ExtractRiskFromRawMessages(rawMessages []tongoWallet.RawMessage) (*Risk, er
 		if err := tlb.Unmarshal(rawMsg.Message, &m); err != nil {
 			return nil, err
 		}
-		tonValue := uint64(m.MessageInternal.Value.Grams)
-		destination, err := ton.AccountIDFromTlb(m.MessageInternal.Dest)
+		var err error
+		risk, err = ExtractRiskFromMessage(m, risk, rawMsg.Mode)
 		if err != nil {
 			return nil, err
 		}
-		risk.Ton += tonValue
-		msgBody := m.MessageInternal.Body.Value.Value
-		if err != nil {
-			continue
+	}
+	return risk, nil
+}
+
+func ExtractRiskFromMessage(m abi.MessageRelaxed, risk *Risk, mode byte) (*Risk, error) {
```

Comment:
I would change this to 
```golang
UpdateRiskFromMessage(risk, message, mode) error
```
making clear that `risk` will be updated in place. 

Otherwise, it's not clear and could lead to an issue later.